src/analysis/special_functions/stirling.lean

```diff
@@ -203,3 +209,88 @@ begin
   rw ←filter.tendsto_add_at_top_iff_nat 1,
   exact tendsto_at_top_cinfi stirling_seq'_antitone ⟨x, hx'⟩,
 end
+
+/-!
+ ### Part 2
+ https://proofwiki.org/wiki/Stirling%27s_Formula#Part_2
+-/
+
+/-- For `n : ℕ`, define `w n` as `2^(4*n) * n!^4 / ((2*n)!^2 * (2*n + 1))` -/
+noncomputable def w (n : ℕ) : ℝ :=
+(2 ^ (4 * n) * n.factorial ^ 4) / ((2 * n).factorial ^ 2 * (2 * n + 1))
```

Comment:
If you `open_locale nat` at the beginning of the file, you can replace all `n.factorial` with `n!` everywhere in this file.

Reply:
done